ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -102,6 +102,33 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-topology-console
```

Comment:
Should we roll these tests into the existing console nightly tests, or at least have a separate dev console job? We would have the same test coverage, but reduce CI costs since we'll be spinning up one cluster instead of three.

Reply:
This is temporary until we increase the timeout of the nightly build